src/plugins/pgcomparator_cdc_operator.py

```diff
@@ -64,11 +71,15 @@ def __init__(
         self.key_column = key_column
         self.use_key = use_key
         self.db_conn_id = db_conn_id if db_conn_id else "AIRFLOW_CONN_POSTGRES_DEFAULT"
+        self.no_deletes = no_deletes
+        self.database_password = database_password
 
     def execute(self, context: Dict[str, Any]) -> None:
         env = Env()
         db_connection, _ = env(self.db_conn_id).split("?")
-        db_url = re.sub(r".*?(?=://)", "pgsql", db_connection, count=1)
+        db_pass = db_connection.split("@")[0].split(":")[2]
```

Comment:
Hier zou je wellicht nog dsnparse voor kunnen gebruiken, is toch wat meer leesbaar dan regexes. Wordt al gebruikt in common.objectstore.

Reply:
Oh, dat (`dsnparse`) zou een hele mooie toevoeging zijn. :+1:

Reply:
Split op `@` is niet cool. Connectie string kan volgende vorm hebben:
```pgsql://test@testhost.com:test_password@test_databasehost.com:666/test_database?conn_params=666```

Reply:
dsnparse werkt goed. Stukje eenvoudiger en overzichtelijker